plugins/logs/commands

```diff
@@ -11,34 +12,60 @@ case "$1" in
     APP_ROOT="$DOKKU_ROOT/$APP"
     COLORS=(36 33 32 35 31)
 
-    if (is_deployed "$APP"); then
+    if ! (is_deployed "$APP"); then
+      echo "Application's container not found"
+      exit 1
+    fi
+
+    shift 2;
+    TEMP=$(getopt -o htqn:p: --long help,tail,quiet,num:,ps: -n 'dokku logs' -- "$@")
+    if [[ $? != 0 ]]; then usage >&2 ; exit 1 ; fi
+    eval set -- "$TEMP"
+
+    LINE_NUMBERS="100"
+    while true; do
+        case "$1" in
+            -h|--help) usage; exit 0 ;;
```

Comment:
Flags are nice but basically impossible in an app like dokku due to our usage of shell :(